pkg/runtime/docker/client.go

```diff
@@ -49,6 +56,102 @@ func (dc *dockerClient) PullImage(image string) (io.ReadCloser, error) {
 	return dc.client.ImagePull(context.Background(), image, types.ImagePullOptions{})
 }
 
+func (dc *dockerClient) ExportImage(image string) (io.ReadCloser, string, error) {
+	config, err := dc.client.ContainerCreate(context.Background(), &container.Config{
+		Cmd:   []string{"sh"}, // We need a temporary command, this doesn't need to exist in the image
+		Image: image,
+	}, nil, nil, "")
+	if err != nil {
+		return nil, "", err
+	}
+
+	rc, err := dc.client.ContainerExport(context.Background(), config.ID)
+	return rc, config.ID, err
+}
+
+func (dc *dockerClient) RunContainer(image string, config *runtime.ContainerConfig, name string) error {
+	// TODO: WIP transform runtime.ContainerConfig to this, perform the ContainerStart call
+	dc.client.ContainerCreate(context.Background(), &container.Config{
+		AttachStdin:     true,
+		AttachStdout:    true,
+		AttachStderr:    true,
+		ExposedPorts:    nil,
+		Tty:             true,
+		OpenStdin:       true,
+		StdinOnce:       false,
+		Env:             nil,
+		Cmd:             nil,
+		Healthcheck:     nil,
```

Comment:
remove the unset options here to make sure the default values are used, and make it explicit what we set

Reply:
Yes, they were just temporarily there while I figured out which ones to set.